### ISSUE TITLE
notify_progress is not defined

### ISSUE THREAD

**Issue by MonsieurPoutounours** [joshp23/ttrss-to-wallabag-v2#34]
Hi,

I'm facing an error using the plugin after TT-RSS upgrade (last commit of TT-RSS and plugin at this time).

```
ReferenceError: notify_progress is not defined

Stack trace:
postArticleToWallabag@https://rss.poutounours.me/index.php:164:3
onclick@https://rss.poutounours.me/index.php#f=-3&c=0:1:1
```

**Comment by titul** [comment]
> Hi,
> 
> I'm facing an error using the plugin after TT-RSS upgrade (last commit of TT-RSS and plugin at this time).
> 
> ```
> ReferenceError: notify_progress is not defined
> 
> Stack trace:
> postArticleToWallabag@https://rss.poutounours.me/index.php:164:3
> onclick@https://rss.poutounours.me/index.php#f=-3&c=0:1:1
> ```

Hello,

Replace 'Notify.' by 'notify_' in file wallabag_v2/wallabag_v2.js for ttrss v18.12.

Bye

**Comment by joshp23** [comment]
@MonsieurPoutounours, please see issues #30 and #32 and indeed #33. This problem was addressed with f586edb4dce9f60c458152b823b029c76b2e9ed6, so all you need to do is update this plugin to the [latest release](https://github.com/joshp23/ttrss-to-wallabag-v2/releases/latest).
